### DISCUSSION TITLE
.sy and .sl SyGuS file extensions.

### DISCUSSION THREAD

**Discussion by pauljurczak** [cvc5/cvc5#9812] [General]
Is there any difference between .sy and .sl SyGuS file extensions? I've seen both used. I recently installed SyGuS Language extension for VSCode, and it looks for .sl file extension.

**Comment by ajreynol** [comment]
In cvc5, there isn't a difference.

I believe *.sl is the agreed upon standard extension, although historically we had used *.sy originally.
